internal/controller/numaflowcontrollerrollout_controller.go

```diff
@@ -607,8 +639,11 @@ func getDeploymentCondition(status appsv1.DeploymentStatus, condType appsv1.Depl
 // - whether it exists
 // - error if any
 func (r *NumaflowControllerRolloutReconciler) getNumaflowControllerDeployment(ctx context.Context, controllerRollout *apiv1.NumaflowControllerRollout) (*appsv1.Deployment, bool, error) {
+	instaceID := controllerRollout.Spec.Controller.InstanceID
+	numaflowControllerDeploymentInstanceName := fmt.Sprintf("%s%s", NumaflowControllerDeploymentName, instaceID)
```

Comment:
nit: spelling of "instaceID"

Reply:
you have no idea how many times I misspelled that today and still missed some, thanks for catching this